<launch>
    <param name="BA" type="bool" value="true" />
    <param name="BA_minimum_sliding_window_size" type="int" value="10"/>
    <param name="BA_sliding_window_size" type="int" value="30"/>
    <param name="BA_Levenberg_Marquardt_coefficient" type="double" value="0.0"/>
    <param name="save_directory" type="string" value="/home/jacko/Documents/PlaneLO_ws/src/07_mapped"/>
    
    <node pkg="planelo" type="demo" name="demo" output="screen" />
    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find planelo)/rviz_cfg/demo.rviz" />
    </group>

</launch>
